01_fib/kaizhu/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fib(7)
+}
+
+func fib(n int64) {
+
+	switch {
+	// print 0 if n is 0
```

Comment:
Don't add comments for self-evident code. Ditto below.

Reply:
Sure.